src/main/java/io/fabric8/maven/docker/log/SharedPrintStream.java

```diff
@@ -0,0 +1,35 @@
+package io.fabric8.maven.docker.log;
+
+import java.io.PrintStream;
+
+class SharedPrintStream {
+    private PrintStream printStream;
+
+    private int numUsers;
+    
+    public SharedPrintStream(PrintStream ps) {
+        this.printStream = ps;
+        this.numUsers = 1;
+    }
+
+    public PrintStream getPrintStream() {
+        return printStream;
+    }
+
+    public void setPrintStream(PrintStream printStream) {
```

Comment:
Doesnt look like that this method is ever used. Also, removing this set-method makes the class immutable with respect to the printstream which is a good thing :)